loading.bs

```diff
@@ -146,6 +146,10 @@ spec: http-dig-alg; urlPrefix: https://www.iana.org/assignments/http-dig-alg/htt
 spec: SHA2; urlPrefix: http://csrc.nist.gov/publications/fips/fips180-4/fips-180-4.pdf
     type: dfn
         text: SHA-256; url: #
+spec: network-error-logging; urlPrefix: https://w3c.github.io/network-error-logging/#
+    type: dfn
+        text: generate a network error report ; url: generate-a-network-error-report
```

Comment:
Humm.. when I removed this line I saw the error even after updating the bikeshed data.
```
LINE ~1059: No 'dfn' refs found for 'generate a network error report'
```

Reply:
@tabatkins, did I do something wrong? https://github.com/tabatkins/bikeshed-data/commit/913e4ff6ee06e56f604f53a6906c729576fe0c6c incorporated the network-error-logging data but doesn't include this definition. Do I need `data-export=""` instead of just `data-export`?

Reply:
Checking with @plinss about this; the only difference from how Bikeshed normally outputs heading-dfns is that Bikeshed includes a `data-lt`, and it might be that Shepherd (@plinss' tool that parses the specs) relies on the presence of that attribute.

(Boolean vs empty value on `data-export` doesn't matter.)

Reply:
And fixed, it was a minor bug in Shepherd's scraping logic for ReSpec documents; it was looking for the definition attributes on the *section* rather than the heading (since respec is section-centric). Fixed now, and the definition shows up as of <https://github.com/tabatkins/bikeshed-data/commit/e62a91cf098bc6f293a711660205d5d05cf2e781>